conductor/lib/common.py

```diff
@@ -416,6 +416,15 @@ def __init__(self):
 
         if 'auth_url' not in combined_config:
             combined_config['auth_url'] = 'https://dashboard.conductortech.com'
+        
+        if 'api_key' in combined_config:
+            try:
+                json_key = json.loads(combined_config['api_key'].replace("\n", ""))
```

Comment:
Also replace `\r` for Windows line breaks.

Reply:
Done.